src/components/LandingPage.vue

```diff
@@ -1904,17 +1904,20 @@ end`,
     backup() {
       this.config.wowpath.versions.forEach((version, versionindex) => {
         version.accounts.forEach((account, accountindex) => {
-          backupIfRequired(
-            this.WeakAurasSaved(version.name, account.name),
-            this.config.backup,
-            account.savedvariableSize,
-            `${version.name}#${account.name}`,
-            (fileSize) => {
-              this.config.wowpath.versions[versionindex].accounts[
-                accountindex
-              ].savedvariableSize = fileSize;
-            }
-          );
+          this.addonsInstalled.forEach((addon) => {
+            backupIfRequired(
+              addon.svPathFunction(version.name, account.name),
+              this.config.backup,
+              account.savedvariableSizeForAddon[addon.addonName],
+              `${version.name}#${account.name}`,
+              (fileSize) => {
+                this.config.wowpath.versions[versionindex].accounts[
+                  accountindex
+                ].savedvariableSizeForAddon[addon.addonName] = fileSize;
```

Comment:
For whatever reason this is not saved over restarts. Can someone give me a hint?